allennlp/service/predictors/predictor.py

```diff
@@ -15,12 +17,28 @@ def __init__(self, model: Model, dataset_reader: DatasetReader) -> None:
         self._model = model
         self._dataset_reader = dataset_reader
 
+    @staticmethod
+    def load_line(line: str) -> JsonDict:
```

Comment:
Do we really want this to be static?  I'm guessing you did this because pylint complained - I'd probably just ignore pylint on a method like this that's meant to be overridden.  Making it static is limiting to future subclasses.

Reply:
changed it to a regular instance method and added a pylint ignore